config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -50,6 +50,10 @@ presubmits:
         envs:
         - name: KUBE_FORCE_VERIFY_CHECKS
           value: "n"
+        - name: EXCLUDE_TYPECHECK
+          value: "y"
+        - name: EXCLUDE_GODEP
+          value: "y"
```

Comment:
also skipping the checks in the godep job which we missed earlier :-) cc @cblecker